app/cafe.py

```diff
@@ -0,0 +1,24 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError, NotWearingMaskError, OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends must be vaccinated.")
+
+        if visitor.get("vaccine").get("expiration_date") < date.today():
+            raise OutdatedVaccineError(
+                "One of the friends has an expired vaccine."
```

Comment:
You are not sure if there is just one friend with expired vaccine, fix error message